### PR TITLE
docs(android/app): Add help page on using the banner

### LINKED ISSUE #10629: docs(android): update banner documentation
We need to refresh any references to the banner and document the usage of it with the changes in v17.

See also #10630.

### PR DESCRIPTION
**Pull request by darcywong00**
Fixes #10629  and follows #10686 on updating the Keyman for Android in-app help

This adds a help page about using the banner

Also refreshes the phone-layout screenshots for the Settings help

## User Testing

**Setup** - Install the PR build of Keyman for Android

* **TEST_INAPP_HELP** - Verifies navigation of help page about the using the banner

1. Launch Keyman for Android and dismiss the "Get Started" menu
2. From Keyman Settings --> Info
3. On the main Keyman for Android help page, verify the "Using Keyman for Android" section has a page "Using the Banner on the Keyboard"
4. Select "Using the Banner on the Keyboard"
5. Verify a page appears about using the banner on the keyboard

### CODE REVIEW

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_INAPP_HELP (PASSED):** Tested with the attached PR build (17.0.264-alpha-test-10691) in an Android 12 / API 31 emulator and here are my observations: 1. Opened the Info page. 2. Verified the "Using Keyman for Android" section has a page "using the Banner on the Keyboard". 3. Clicked on the link for "Using the Banner on the Keyboard" and confirmed that a relevant page displayed, detailing the functionality of the banner on the keyboard. Seems to be working as expected.

File: android/help/basic/using-the-banner.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+## Using the Suggestion Banner
+
+If a [dictionary is installed](installing-dictionaries) and enabled for the active Keyman keyboard, the banner will display suggestions that can be selected.
+
+![](../android_images/settings-suggestions-ap.png)
+
+* The banner can slide when a touch is held down to reveal more suggestions
+* Overly-long suggestions are partially hidden, but expand when touched
+* If a suggestion is selected while attempting to scroll the banner, it will not be applied
```
Comment (mcdurdin) [inline android/help/basic/using-the-banner.md:21]:
```suggestion
* A suggestion will not be applied while you slide the banner
```

File: android/help/basic/using-the-banner.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* Display a Keyman-themed banner so popups and gestures for the top row of keys are visible
+* Reserved for future functionality
+
+## Using the Suggestion Banner
+
+If a [dictionary is installed](installing-dictionaries) and enabled for the active Keyman keyboard, the banner will display suggestions that can be selected.
+
+![](../android_images/settings-suggestions-ap.png)
+
+* The banner can slide when a touch is held down to reveal more suggestions
```
Comment (mcdurdin) [inline android/help/basic/using-the-banner.md:19]:
```suggestion
* You can slide the banner to see more suggestions by dragging it horizontally with your finger
```

File: android/help/basic/using-the-banner.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+* Overly-long suggestions are partially hidden, but expand when touched
```
Comment (mcdurdin) [inline android/help/basic/using-the-banner.md:20]:
```suggestion
* Overly-long suggestions are partially hidden, but expand when you hold your finger on them
```

**Review by mcdurdin — APPROVED** [review]
We probably need to get consistent on our touch terminology -- I've made suggestions on the banner gestures but I am not sure they are 'better'

**Comment by mcdurdin** [comment]
gitguardian had a false positive here. Deleted because it was just noise

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.269-beta](https://keyman.com/downloads/releases/beta/17.0.269)

**Pull request merged**